### DISCUSSION TITLE
Brasil Track Panels

### DISCUSSION THREAD

**Discussion by pachicodes** [gittogethers/octogatosconf22#13] [General]
## Github Education
Host: @mlarasusan
Guests: Otacilio, Lavinia , Aaron, Germano

## Comunities
Host: @pachicodes
Guests: Lissa Ferreira, Gisele Passuti, Beatriz Oliveira and Larissa Vitoriano

## Github Stars
Host: @pachicodes
Guests: Mario Souto, Leticia Leonardo, Julio Arruda, Willian Justen

**Comment by pachicodes** [comment]
@AndreaGriffiths11

**Reply by AndreaGriffiths11** [reply]
Thank you so much Pachi! I'll create the assets and share them here; also I'll make sure they are all on the website
